helm/reana/README.md

```diff
@@ -19,6 +19,7 @@ This Helm automatically prefixes all names using the release name to avoid colli
 | `components.reana_ui.image`                              | [REANA-UI image](https://hub.docker.com/repository/docker/reanahub/reana-ui) to use  | `reanahub/reana-ui:<chart-release-verion>`      |
 | `components.reana_ui.imagePullPolicy`                    | REANA-UI image pull policy                                                           | IfNotPresent                                    |
 | `components.reana_ui.pooling_secs`                       | Frecuency of workflow list page reload in seconds                                    | 15                                              |
+| `components.reana_ui.client_pyenv`                       | REANA python environment                                                             | None                                            |
```

Comment:
- For the help text, it is important to say "client". Perhaps say "REANA-Client python environment to source in the welcome example."  That should be sufficiently descriptive...

- While you are at it, can you also fix typos in the above lines e.g. `frecuency -> frequency"..... I see also "verion" instead of "version"... Would be cool.